eng/common/TestResources/New-TestResources-Wrapper.ps1

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+param(
+[string] $ServiceDirectory,
+[string] $BaseName
+)
+$run = Read-Host "The test resources needed to run the live tests could not be located.`nWould you like to run the resource creation script? [y/n]"
```

Comment:
Is this wrapper even useful? Is it only so you can do the prompting?

Reply:
Yes, it is only for the prompting.
I'm not aware of a way to provide an interactive prompt within the test run context.

Reply:
If you are not opposed to adding this directly to New-TestResources along with a new switch param, I can do that instead.

Reply:
I like keeping it separate for now and I also like it is in a .NET specific script instead of under eng/common.